### PR TITLE
New README.md

### PR DESCRIPTION
**Pull request by Ryvix**
Hopefully I did this right.

This is an update to the README.md file.

I added some steps on using the Generator but now I'm not sure if that should even go in here or in the Generator README.md. It seemed useful to know how to use WPBP properly though so I left it there.

I added descriptions and links to all the things in the wpbp.json file.

It's still missing some info under the Snippets section. For example it probably could use a description under the Snippets heading about what snippets actually are and where to find them. It also has some missing descriptions for some and there are no links to them since I don't know where or even what exactly they are. Maybe that will come when I get more experience using it all.

I sort of merged what I had before into the existing README.md so of course feel free to edit it however you like it. I'm not sure what parts you want to add/remove/replace, etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,57 +7,179 @@
...
+> :warning: These steps may be incomplete and need expanded upon and testing. For example, you may also want to run
+> tests, Codeception, etc.
+
+1. `wpbp-generator --json` - Generate a new wpbp.json file in the current folder.
+2. Edit the wpbp.json to your liking by removing what you don't want.
+3. `wpbp-generator` - Download WPBP and extract it to a new folder and removes the parts you don't want.   
+   Your new plugin folder is created as a subfolder in the current working directory.
+4. `cd your-plugin-folder` - Change into your plugin folder:
+5. Edit composer.json and change versions or packages to your liking.
+6.
```
Comment (Mte90) [inline README.md:57]: This is rendered badly as the subitem is not indented right.

**Comment by Mte90** [comment]
So I like the new version, is very better as explanation.  

My doubts are just for the library list. I chose the table version to get a readme that is more shorter but in that way it is difficult to get the description. A solution could be use the table just to have 2 columns with a single row with all the descriptions.
For snippets they are in the boilerplate but points the position to the file can be a problem as it will means update the readme at every release to check if the line is changed. Reading your readme version I think that is complete as explanation.

About CMB2 as columns yes but in some cases you just want that library and not the whole CMB2.

File: README.md [hunk 2]
```diff
@@ -7,57 +7,179 @@
 [![Backers on Open Collective](https://opencollective.com/WordPress-Plugin-Boilerplate-Powered/backers/badge.svg)](#backers)
 [![Sponsors on Open Collective](https://opencollective.com/WordPress-Plugin-Boilerplate-Powered/sponsors/badge.svg)](#sponsors)
```
Comment (erikyo) [inline README.md:8]: would be cool to show the build / test status like
```suggestion
<div>
  [![Backers on Open Collective](https://opencollective.com/WordPress-Plugin-Boilerplate-Powered/backers/badge.svg)](#backers)
  [![Sponsors on Open Collective](https://opencollective.com/WordPress-Plugin-Boilerplate-Powered/sponsors/badge.svg)](#sponsors)
  <a href="https://github.com/WPBP/WordPress-Plugin-Boilerplate-Powered/actions">
    <img alt="build" src="https://img.shields.io/github/actions/workflow/status/WPBP/WordPress-Plugin-Boilerplate-Powered/integrate.yml" />
  </a>
  <a href="https://github.com/WPBP/WordPress-Plugin-Boilerplate-Powered/actions">
    <img alt="workflows" src="https://github.com/WPBP/WordPress-Plugin-Boilerplate-Powered/actions/workflows/integrate.yml/badge.svg" />
  </a>
</div>
```

**Comment by Mte90** [comment]
I think that now is perfect, what do you think @erikyo @Ryvix?

In case if you want to help further the project a review on https://wpbp.github.io/ can be interesting with a ticket on https://github.com/wpbp/wpbp.github.io

**Comment by erikyo** [comment]
LGTM! 🚀

**Comment by Ryvix** [comment]
If you like it then I'm happy too. Nice work 😀

**Pull request merged**